pages/installation/virtual_machine_appliances.rst

```diff
@@ -35,6 +35,11 @@ completed:
 
 .. image:: /images/virtualbox3.png
 
+
+Note:
+If you don't have a working DHCP server for your virtual machine, you will get the error message "Your appliance came up without a configured IP address. Graylog is probable not running correctly!". 
```

Comment:
The message should be better visible, e. g. by using a quoted block: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#quoted-literal-blocks